back/config/schemas/settings.schema.json.erb

```diff
@@ -1059,6 +1059,18 @@
           "allowed": { "type": "boolean", "default": true },
           "enabled": { "type": "boolean", "default": false }
         }
+      },
```

Comment:
@sebastienhoorens I've left this as it is despite Koen's suggestions, given that this seems to be the way we've done it everywhere else